pandas-stubs/_typing.pyi

```diff
@@ -576,6 +576,9 @@ S2 = TypeVar(
     | list[str],
 )
 
+class UnknownSeries(Series[Any]): ...
+class UnknownIndex(Index[Any]): ...
```

Comment:
I think these should move to `core/series.pyi` and `core/indexes/base.pyi`, respectively